checks.d/tokumx.py

```diff
@@ -424,7 +424,11 @@ def collect_metrics(self, instance, server, conn, db, tags):
                 db_tags = list(tags)
                 db_tags.append('db:%s' % dbname)
                 db = conn[dbname]
-                stats = db.command('dbstats')
+                try:
+                    stats = db.command('dbstats')
+                except:
```

Comment:
Is there a more specific exception that can be caught ? 
It's usually bad practice to have bare `except:` statements:
https://docs.python.org/2/howto/doanddont.html#except